PyInstaller/compat.py

```diff
@@ -889,3 +891,30 @@ def check_requirements():
             raise SystemExit('PyInstaller cannot check for assembly dependencies.\n'
                              'Please install PyWin32.\n\n'
                              'pip install pypiwin32\n')
+
+
+@contextmanager
+def open_pysrc(filename):
+    if getattr(tokenize, 'open', None) is not None:
+        print('USING tokenize')
```

Comment:
Please remove this line. I you feel this is really necessary, it should be a log message of level DEBUG and be much more spelling.